portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Same for blog.html. You added it in two commits.

Reply:
Same as portfolio.html

Reply:
Fixed it.